public/app/features/alerting/unified/components/rule-viewer/v2/RuleContext.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+import { CombinedRule, RuleIdentifier } from 'app/types/unified-alerting';
+
+interface Context {
+  rule: CombinedRule;
+  identifier: RuleIdentifier;
+}
+
+const AlertRuleContext = React.createContext<Context | undefined>(undefined);
+
+type Props = Context & React.PropsWithChildren & {};
```

Comment:
what is the value of adding this empty object? in the type?

Reply:
None, actually! We don't have any props to add yet, so it serves as a placeholder for when we need them.